patient-care/webapp/client/shared/editSampleGroup.js

```diff
@@ -51,6 +57,46 @@ Template.editSampleGroup.onRendered(function () {
 
   instance.$(".sample-group-version").popup({
     position : "bottom right",
+    content: "Sample group version",
+  });
+
+  instance.$(".data-set-search").search({
+    apiSettings: {
+      url: `${location.origin}/search/data-sets` +
+          `?token=${Accounts._storedLoginToken()}&q={query}`,
+      onResponse(response) {
+        // modify the response to remove existing data sets
```

Comment:
In a later pull, presumably. This is annoying but shouldn't block this pull from going to prod.